xmtp/src/utils.rs

```diff
@@ -21,6 +21,10 @@ pub fn build_user_invite_topic(public_key: String) -> String {
     format!("xmtp/1/invite-{}", public_key)
 }
 
+pub fn build_installation_message_topic(installation_id: &str) -> String {
+    format!("xmtp/1/message-{}", installation_id)
```

Comment:
I just realized none of our V3 topics have the `/proto` suffix, which all the existing (v1/v2) topics do. We should decide if we care

Reply:
Mostly because Waku had the suffix, but also I guess to allow for non-protobuf messages in the future.